packages/squiggle-lang/src/rescript/ReducerInterface/ReducerInterface_GenericDistribution.res

```diff
@@ -233,6 +248,21 @@ let dispatchToGenericOutput = (call: ExpressionValue.functionCall, _environment)
   | ("normalize", [EvDistribution(dist)]) => Helpers.toDistFn(Normalize, dist)
   | ("klDivergence", [EvDistribution(a), EvDistribution(b)]) =>
     Some(runGenericOperation(FromDist(ToScore(KLDivergence(b)), a)))
+  | ("logScore", [EvDistribution(prior), EvDistribution(prediction), EvNumber(answer)])
+  | (
+    "logScore",
+    [EvDistribution(prior), EvDistribution(prediction), EvDistribution(Symbolic(#Float(answer)))],
+  ) =>
+    Some(runGenericOperation(FromDist(ToScore(LogScore(prediction, answer)), prior)))
+  | ("logScoreAgainstImproperPrior", [EvDistribution(prediction), EvNumber(answer)])
```

Comment:
why is it an ``improper`` prior?

Reply:
Because it's the same score as the one with a normal prior, except the prior is 1 everywhere. So instead of log(answer(x)/prior(x)), you just have log(answer(x)/1)

Reply:
This is confusing.

How about
"logScoreWithPrior" and "logScore". 

I believe logscore is often used without a formal prior.

We could also just have ``logScore`` with an optional ``prior`` paramater. Like,
``logScore({prior, prediction, answer})``